ci/Jenkinsfile-kubernetes

```diff
@@ -0,0 +1,123 @@
+#!/usr/bin/groovy
+@Library('test-shared-library') _
+
+properties(
+        [
+                pipelineTriggers([cron('H 16 * * *')]),
+                buildDiscarder(logRotator(numToKeepStr: '10'))
+        ]
+)
+
+def getKubernetesSparkVersions() {
+    def versionLine = readFile("gradle.properties").split("\n").find() { line -> line.startsWith('supportedSparkVersions') }
```

Comment:
Is it possible to read the `gradle.properties` file once? E.g convert to a map first and then use it?

Reply:
Good idea, done. I re-triggered the job to verify it works after this change